lib/praxis-blueprints/blueprint.rb

```diff
@@ -36,37 +55,33 @@ def self.inherited(klass)
       super
 
       klass.instance_eval do
-        @views = {}
         @options = {}
         @domain_model = Object
+        @default_fieldset = {}
       end
     end
 
     # Override default new behavior to support memoized creation through an IdentityMap
-    def self.new(object, decorators = nil)
-      if @@caching_enabled && decorators.nil?
-        cache = if object.respond_to?(:identity_map) && object.identity_map
-                  object.identity_map.blueprint_cache[self]
-                else
-                  self.cache
-                end
-
-        return cache[object] ||= begin
+    def self.new(object)
+      # TODO: do we want to allow the identity map thing in the object?...maybe not.
+      if @@caching_enabled
+        return self.cache[object] ||= begin
           blueprint = self.allocate
-          blueprint.send(:initialize, object, decorators)
+          blueprint.send(:initialize, object)
           blueprint
         end
       end
 
       blueprint = self.allocate
-      blueprint.send(:initialize, object, decorators)
+      blueprint.send(:initialize, object)
       blueprint
     end
 
     def self.family
       'hash'
     end
 
+    # TODO: should we get rid of shallow? (or describe in general?)
```

Comment:
I think we use describe it in the docs. Not sure if we need shallow anymore though or not, good question

Reply:
we "used" to use them in the Praxis docs...but now we moved to OpenAPI docs and we technically do not "need" describe anymore :)
I was pseudo-hesitant to remove it all...but maybe we just do :)
Attributor will still have all the describes, but...we can remove that later.

Reply:
Then let's just remove it.